docker/config/webapp.env

```diff
@@ -3,6 +3,7 @@ DJANGO_DEV=True
 DJANGO_DEBUG=True
 DATABASE_URL=postgres://pontoon:asdf@postgresql/pontoon
 SESSION_COOKIE_SECURE=False
+CSRF_COOKIE_SECURE=False
```

Comment:
It was displayed together with `SESSION_COOKIE_SECURE` in doc and that one was False by default in settings. Should I change that to True or make both True by default?